src/ontology/mondo-edit.obo

```diff
@@ -94485,7 +94485,7 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0004813
 name: tuberculous pneumothorax
-def: "A pneumothorax in which air enters into the pleural cavity." [DOID:9534, http://books.google.com/books?id=9QcpAAAAYAAJ&pg=PA492&lpg#v=onepage&q=&f=false]
+def: "A pneumothorax caused by tuberculosis infection, typically resulting from rupture of a cavitary tuberculous lesion into the pleural space." [DOID:9534, https://orcid.org/0000-0003-2955-4640, PMID:21255490]
```

Comment:
I think we can remove the DO ID for this definition. 
The current definition is different enough from the DO definition, so it doesn't make sense to refer back to them anymore.